ERCS/erc-7673.md

```diff
@@ -0,0 +1,373 @@
+---
+eip: 7673
+title: Distinguishable base256emoji Addresses
+description: Depict Account Addresses As A String of 20 Emoji
+author: William Morriss (@wjmelements)
+discussions-to: https://ethereum-magicians.org/t/erc-7673-distinguishable-account-addresses/19461
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-04-01
+---
+
+## Abstract
+
+Immediately replace the hexadecimal address encoding with base256emoji in all user interfaces.
```

Comment:
Perhaps more of a recommendation for the primary display and use of addresses?

Reply:
The current display is [not sufficiently idiot-proof](https://ethereum-magicians.org/t/patch-proposals-for-the-ongoing-wallet-homomoroph-attacks/16686/9?u=wjmelements) and is causing losses so ERC-55 should be replaced as soon as possible. The rationale is similar to ERC-55 so I would copy the style of that abstract but that ERC does not have an abstract.